src/VisualStudio/IntegrationTests/CSharpInteractiveDemo.cs

```diff
@@ -0,0 +1,111 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using Roslyn.VisualStudio.Test.Utilities;
+using Xunit;
+
+namespace Roslyn.VisualStudio.Integration.UnitTests
+{
+    [Collection(nameof(SharedIntegrationHost))]
+    public class CSharpInteractiveDemo
+    {
+        private IntegrationHost _host;
+        private InteractiveWindow _interactiveWindow;
+
+        public CSharpInteractiveDemo(IntegrationHost host)
+        {
+            _host = host;
+
+            host.Initialize();
+
+            _interactiveWindow = host.CSharpInteractiveWindow;
+
+            _interactiveWindow.Show();
+            _interactiveWindow.Reset();
+
+            _interactiveWindow.SubmitTextToRepl(@"#r ""WindowsBase""
+#r ""PresentationCore""
+#r ""PresentationFramework""
+#r ""System.Xaml""", timeout: 20000);
+
+            _interactiveWindow.SubmitTextToRepl(@"using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;");
+        }
+
+        [Fact]
+        public void BclMathCall()
+        {
+            _interactiveWindow.SubmitTextToRepl("Math.Sin(1)");
+            Assert.True(_interactiveWindow.CheckLastReplOutputEquals("0.8414709848078965"));
+        }
+
+        [Fact]
+        public void BclConsoleCall()
+        {
+            _interactiveWindow.SubmitTextToRepl(@"Console.WriteLine(""Hello World"");");
+            Assert.True(_interactiveWindow.CheckLastReplOutputEquals("Hello World"));
+        }
+
+        [Fact]
+        public void ForStatement()
+        {
+            IntegrationLog.BeginScenario("");
+
+            _interactiveWindow.SubmitTextToRepl("for (int i = 0; i < 10; i++) Console.WriteLine(i * i);");
+            Assert.True(_interactiveWindow.CheckLastReplOutputEndsWith("81"));
+
+            IntegrationLog.EndScenario();
+        }
+
+        [Fact]
+        public void ForEachStatement()
+        {
+            _interactiveWindow.SubmitTextToRepl(@"foreach (var f in System.IO.Directory.GetFiles(@""c:\windows"")) Console.WriteLine($""{f}"".ToLower());");
```

Comment:
This test assumes that windows is installed on `C:`.  That will not be the case for all machines.